lib/rules/no-extra-semicolons/__tests__/index.js

```diff
@@ -555,6 +555,6 @@ testRule(rule, {
     code: "a { .mixin();\ncolor: red;; }",
     message: messages.rejected,
     line: 2,
-    column: 10,
+    column: 11,
```

Comment:
I actually believe that the old column here was incorrect, `11` makes more sense when checking it in my editor for example (the editor is reporting `11` as the first semicolon after `red`. 

I'm happy to be proved wrong though!